GlobalScaling/GlobalScalingTests.c

```diff
@@ -0,0 +1,205 @@
+#include "GlobalScalingTests.h"
+#include "GlobalScaling.h"
+
+void callAllTestsGroup2(){
+    cleanTest();
+
+    dirtyTest1();
+
+    dirtyTest2();
+
+    dirtyTest3();
+
+    dirtyTest4();
+
+    dirtyTest5(); 
+}
+
+/*
+    -First clean test
+    -intializes each index of the matrix to the value of 1
+    -calls globalscaling
+    -checks each index if the correct value was entered at that index
+    -if unsuccessful prints test failed
+*/
+void cleanTest(){
+    for(int i = 0; i < 4; i++){
+        for(int j = 0; j < 4; j++){
+            transformationMatrix[i][j] = 1.0;
+        }
+    }
+    printf("*************************Running clean test****************************");
+    float check = 1.0;
+    globalScaling();
+    for(int i = 0; i < 4; i++){
+        for(int j = 0; j < 4; j++){
+            if(transformationMatrix[i][j] != check){
+                printf("\n");
+                printf("Test failed\n");
```

Comment:
Because we have multiple groups performing tests, can you change this to say "Global Scaling: <test name> failed\n", where <test name> is the name of this function